pkg/collector/corechecks/loader.go

```diff
@@ -52,6 +52,11 @@ func NewGoCheckLoader() (*GoCheckLoader, error) {
 	return &GoCheckLoader{}, nil
 }
 
+// Name return returns Go loader name
+func (gl *GoCheckLoader) Name() string {
+	return "core"
```

Comment:
Open question: `GoCheckLoader` should use `go` or `core` as name ?

Reply:
+1 for `core`